iam/api/QuickStartV2Test/runTests.ps1

```diff
@@ -0,0 +1,17 @@
+# Copyright(c) 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+import-module -DisableNameChecking ..\..\..\BuildTools.psm1
+
+dotnet restore
```

Comment:
Can you include the junit test logger here? Take a look at the runTests.ps1 in the KMS samples to see how it's included.

Reply:
Done, I think? lmk if I didn't do it right.